library/httpd

```diff
@@ -11,5 +11,5 @@ Directory: 2.4
 
 Tags: 2.4.46-alpine, 2.4-alpine, 2-alpine, alpine
```

Comment:
I wonder if we should add some more specific aliases (even though we only support one version)?

Reply:
:man_shrugging: we've really only done that for languages and none of our "service" images have Alpine versions in their tags:  (`bash`, `docker` , `haproxy`, `memcached`, `postgres`, `redis`).

(Maybe `bash` needs it?)